### PR TITLE
[WIP] Pr add zuul jobs trial and error

### PR DESCRIPTION
**Pull request by sbueringer**
just another PR to play around with Zuul. Won't be merged ever. Please ignore it :)

### CODE REVIEW

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
> Build failed.
> 
> * cluster-api-provider-openstack-current-acceptance-test-v1.17  : RETRY_LIMIT in 1h 41m 53s
> * [cluster-api-provider-openstack-current-acceptance-test-v1.18 ](https://logs.openlabtesting.org/logs/24/524/56af693b5fc9b42c268246056bc87b968899377f/check/cluster-api-provider-openstack-current-acceptance-test-v1.18/c5694cc/) : SUCCESS in 52m 36s
> * [cluster-api-provider-openstack-stable-acceptance-test-master-pr ](https://logs.openlabtesting.org/logs/24/524/56af693b5fc9b42c268246056bc87b968899377f/check/cluster-api-provider-openstack-stable-acceptance-test-master-pr/5074b28/) : SUCCESS in 40m 02s

@wangxiyuan do you know what triggers retries and how I can debug them?

**Comment by wangxiyuan** [comment]
just comment `recheck` to run the job again.

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
@wangxiyuan That's what I found at the end of a job which was then retried:
```
2020-03-26 19:05:07.391009 | ubuntu-bionic-large | Deleting cluster...
2020-03-26 19:05:07.900024 | ubuntu-bionic-large | kubectl delete machinedeployment --all --ignore-not-found
2020-03-26 19:05:09.615753 | ubuntu-bionic-large | machinedeployment.cluster.x-k8s.io "capi-quickstart-md-0" deleted
2020-03-26 19:05:09.667357 | ubuntu-bionic-large | kubectl delete kubeadmcontrolplane --all --ignore-not-found
2020-03-26 19:05:09.984949 | ubuntu-bionic-large | kubeadmcontrolplane.controlplane.cluster.x-k8s.io "capi-quickstart-control-plane" deleted
2020-03-26 19:10:12.290689 | [Zuul] Log Stream did not terminate
2020-03-26 19:10:12.292339 | ubuntu-bionic-large | ERROR
2020-03-26 19:10:12.292816 | ubuntu-bionic-large | {
2020-03-26 19:10:12.292978 | ubuntu-bionic-large |   "msg": "SSH Error: data could not be sent to remote host \"37.153.172.211\". Make sure this host can be reached over ssh",
2020-03-26 19:10:12.293080 | ubuntu-bionic-large |   "unreachable": true
2020-03-26 19:10:12.293226 | ubuntu-bionic-large | }
2020-03-26 19:10:12.342547 | 
2020-03-26 19:10:12.342720 | PLAY RECAP
```

Any idea what's going on? :)

**Comment by wangxiyuan** [comment]
recheck

File: hack/ci/e2e-conformance.sh [hunk 1]
```diff
@@ -419,12 +414,24 @@ main() {
...
+    #run_tests
+  fi
+
+  DELETE_CLUSTER=${DELETE_CLUSTER:-""}
+  if [[ "${DELETE_CLUSTER}" == "yes" || "${DELETE_CLUSTER}" == "1" ]]; then
+    echo "Dumping logs"
+    dump_logs
+
+    echo "Deleting cluster..."
+    delete_cluster
```
Comment (sbueringer) [inline hack/ci/e2e-conformance.sh]: @wangxiyuan that's the call which was stuck lust week

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
recheck
@wangxiyuan looks like glance is broken now :/. Now idea how

**Comment by wangxiyuan** [comment]
recheck

**Comment by wangxiyuan** [comment]
recheck

**Comment by sbueringer** [comment]
@jichenjc I removed swift and cinder because we're not using them. Can you please take another look?

**Comment by sbueringer** [comment]
recheck

**Comment by sbueringer** [comment]
not required anymore

**Pull request closed without merging**